modules/installation-and-upgrade/pages/proxy-container-installation.adoc

```diff
@@ -5,11 +5,11 @@
 ====
 
 ifeval::[{uyuni-content} == true]
-Only {opensuse} Leap 15.3 and newer are supported to be used as container host for {productname} Proxy containers.
+Only {opensuse} Leap 15.3 and newer are supported to be used as a container host for {productname} Proxy containers.
```

Comment:
It might be best to make a small table to display supported versions for container hosts.

Reply:
I added tables for displaying this information.